lib/inquisition/rubocop/runner.rb

```diff
@@ -11,21 +11,39 @@ class Runner < ::Inquisition::Runner
         fatal: :high
       }.freeze
 
-      ::RuboCop::ConfigLoader::DOTFILE = 'config/.rubocop.yml'.freeze
+      USERS_CONFIG_FILE = '.rubocop.yml'.freeze
 
       def call
-        offenses = Inquisition::Rubocop::RuboCopModifiedRunner.new({}, ::RuboCop::ConfigStore.new).run(['.'])
+        offenses = RuboCopModifiedRunner.new(options, config_store).run(['.'])
```

Comment:
It's a little bit strange that we pass the same options to `RuboCopModifiedRunner` and `ConfigStore`